src/common/registry/test/fixtures/key.fixture.ts

```diff
@@ -1,8 +1,8 @@
 import { hexZeroPad } from '@ethersproject/bytes';
 
 export const key = {
-  key: hexZeroPad('0x12', 98),
-  depositSignature: hexZeroPad('0x12', 194),
+  key: hexZeroPad('0x12', 48),
+  depositSignature: hexZeroPad('0x12', 96),
```

Comment:
It would be useful to add this comment here, otherwise, the code might seem a bit not obvious

```js
/**
 * keys 96 = 48byte
 * signatures 192 = 96byte
 * https://github.com/lidofinance/lido-dao/blob/539a0faf33807d04444047d0905dce2b45260dfa/contracts/0.4.24/lib/SigningKeys.sol#L213-L238
 */
export const KEYS_LENGTH = 96;
export const SIGNATURE_LENGTH = 192;
```

Reply:
explanation has been added